cmd/kube-proxy/app/server_others.go

```diff
@@ -91,8 +91,28 @@ func newProxyServer(
 		return nil, fmt.Errorf("unable to register configz: %s", err)
 	}
 
+	// detect IP Family
+	hostname, err := utilnode.GetHostname(config.HostnameOverride)
```

Comment:
This makes it seem like you're saying the `utilnode.GetHostname` call specifically detects the IP family...

Each of the things below here is used for multiple purposes. I don't think labeling the whole section "detect IP family" is really accurate. You could just drop the comment.

Reply:
done